lecture_1/homework/app/controllers/clans_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ClansController < ApplicationController
+    def index
+      begin
+        clans = Clan.all()
+        render json: clans.to_json(), status: 200
+      rescue ActiveRecord::ActiveRecordError => e
```

Comment:
Jeśli chciałbyś sobie troszkę wysuszyć kontrolery z błedów, możesz skorzystać z metody kontrolera `rescue_from`, trochę o tym będzie w następnych wykładach :) Ogółem jest to nieco czystsze rozwiązanie w kwestii łapania błedów